src/api/WorkspaceConfig.ts

```diff
@@ -1,11 +1,18 @@
 interface WorkspaceConfig {
   name: string;
-  services: ServiceConfig[]
+  services: ServiceConfig[];
+  components: ComponentConfig[];
 }
 
 interface ServiceConfig {
-  name: string;
+  serviceName: string;
   displayName: string;
```

Comment:
why do we need both serviceName and displayName?

Reply:
actually we don't really need for now, I will remove it. We will probably need it in future when we will have a user interface to build the config.